### PR TITLE
Fix boolean type casting

### PR DESCRIPTION
**Pull request by ionl**
whereas construct attempt string
we can't see die message :)

### CODE REVIEW

File: src/Runtime/LambdaRuntime.php [hunk 1]
```diff
@@ -38,7 +38,7 @@ final class LambdaRuntime
 
     public static function fromEnvironmentVariable(): self
     {
-        return new self(getenv('AWS_LAMBDA_RUNTIME_API'));
+        return new self((string)getenv('AWS_LAMBDA_RUNTIME_API'));
```
Comment (mnapoli) [inline src/Runtime/LambdaRuntime.php]:
```suggestion
        return new self((string) getenv('AWS_LAMBDA_RUNTIME_API'));
```

to fix the coding style

**Comment by Nyholm** [comment]
Thank you @ionl for this PR. 

I like it, there is only one small issue with the coding standard. Matthieu made a suggestion, could you just commit that suggestion and this PR will be merged.

File: src/Runtime/LambdaRuntime.php (hunk 1 above)
Comment (ionl) [reply src/Runtime/LambdaRuntime.php]: Sorry for the late, you have right for the coding style (phpStorm don't apply it!)
I see it in https://www.php-fig.org/psr/psr-12/
Thank you

Comment (mnapoli) [reply src/Runtime/LambdaRuntime.php]: 👍 

You need to commit the suggestion. To do that, you need to click on "Unresolve conversation", then click on "Commit suggestion".

The other option is that you change the code yourself, commit and push on the branch to update the pull request.

Comment (ionl) [reply src/Runtime/LambdaRuntime.php]: I'm not a regular, thank for precisions ;)

**Comment by mnapoli** [comment]
Thank you!

**Pull request merged**